src/System.Windows.Forms/src/System/Windows/Forms/ErrorProvider.cs

```diff
@@ -1076,11 +1071,7 @@ public void Remove(ControlItem item)
 
                 if (_tipWindow != null)
                 {
-                    NativeMethods.TOOLINFO_T toolInfo = new NativeMethods.TOOLINFO_T();
-                    toolInfo.cbSize = Marshal.SizeOf(toolInfo);
-                    toolInfo.hwnd = Handle;
-                    toolInfo.uId = item.Id;
-                    UnsafeNativeMethods.SendMessage(new HandleRef(_tipWindow, _tipWindow.Handle), NativeMethods.TTM_DELTOOL, 0, toolInfo);
+                    new ComCtl32.ToolInfoWrapper(this, item.Id).SendMessage(_tipWindow, WindowMessages.TTM_DELTOOLW);
```

Comment:
Whilst this change is absolutely correct, it looks inconsistent with other uses.
Could you please rewrite it as follows?

```suggestion
                    var toolInfo = new ComCtl32.ToolInfoWrapper(this, item.Id);
                    toolInfo.SendMessage(_tipWindow, WindowMessages.TTM_DELTOOLW);
```